src/main/metabase/examples.ts

```diff
@@ -1,7 +1,17 @@
+import { find } from 'lodash';
 import { postgresQuote } from '../../shared';
 import { Field, Table } from './types';
 
-type Display = 'table' | 'bar' | 'row' | 'scalar' | 'map'; // Other types TBD.
+// ----------------------------------------------------------------
+// Types & helpers
+// ----------------------------------------------------------------
+
+/** A section for a group of examples. */
+type ExamplesSection = {
```

Comment:
similar to other comment - should we maybe move all the types related to metabase API to `types.ts`

Reply:
This is not exposed to the outer world and is pretty much exclusive to examples, with a reference from a single function in `api`. I don't think it warrants polluting `types`.